### PR TITLE
Features/7919

### PR DESCRIPTION
**Pull request by sslaws**
## Status
In Development - early merge

## Fixes or Implements [AB#7919](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7919) and [AB#7908](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7908)
- [X] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
DB Model for Account Closure [AB#7879](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7879)
DB Model for Updated Terms of Service [AB#7787](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7787)
DB Model for Last Login [AB#7924](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7924)
Refactoring

Batch Job is not complete but will not impact build/deployment.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### Steps to Reproduce:
If this is a bug, please provide details on how to reproduce the code.

### UI Changes
NO

### Browsers Tested
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments
Early merge as others are waiting on the db model changes.

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_348)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_348)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_348)

File: Apps/Database/src/Delegates/DBProfileDelegate.cs [hunk 1]
```diff
@@ -103,5 +105,20 @@ public DBResult<UserProfile> GetUserProfile(string hdId)
             this.logger.LogDebug($"Finished getting user profile from DB. {JsonSerializer.Serialize(result)}");
             return result;
         }
+
+        /// <inheritdoc />
+        public DBResult<List<UserProfile>> GetAllUserProfilesAfter(DateTime afterDate, int page, int pagesize = 500)
+        {
```
Comment (tiagograf-AOT) [inline Apps/Database/src/Delegates/DBProfileDelegate.cs]: NIT: No trace logs!

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_348)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_348)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_348)

**Pull request merged**